src/current/v24.1/show-virtual-cluster.md

```diff
@@ -49,8 +49,19 @@ This table lists all possible responses from the different `SHOW VIRTUAL CLUSTER
 
 {% include {{ page.version.version }}/physical-replication/show-virtual-cluster-responses.md %}
 
+{{site.data.alerts.callout_success}}
+To find the job ID for the replication stream, use the [`SHOW JOBS`]({% link {{ page.version.version }}/show-jobs.md %}) statement. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM [SHOW JOBS] WHERE job_type = 'REPLICATION STREAM INGESTION';
```

Comment:
Could there be multiple `REPLICATION STREAM INGESTION` jobs? If not (there is only 1), could the statement be more specific:

```suggestion
SELECT job_id FROM [SHOW JOBS] WHERE job_type = 'REPLICATION STREAM INGESTION';
```

Reply:
There potentially could be yes